src/Common/tests/System/Threading/Tasks/TaskAPMExtensions.cs

```diff
@@ -6,8 +6,6 @@
 /// Task APM helper based on http://blogs.msdn.com/b/pfxteam/archive/2011/06/27/10179452.aspx
 /// </summary>
 
-// TODO: #3144 - Remove this class and rewrite tests to work directly with TPL.
```

Comment:
We should still be able to delete this helper, as we have a production version the tests can also use:
https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Threading/Tasks/TaskToApm.cs